dashboard/src/modules/Dashboard/Templates.vue

```diff
@@ -44,31 +28,21 @@
       </v-card>
     </div>
 
-    <div
-      tabindex="-1"
-      class="v-stepper__step subtitle-2 v-stepper__step--active"
-    >
+    <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done